### PR TITLE
Added Meditation Feature (Views + Flow) and updated onboarding + login pages

### PR DESCRIPTION
**Pull request by dguo8412**
# *Added Meditation Feature (Views + Flow) and updated onboarding + login pages*

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
**Issues with Balance logo rendering** 

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*
**ASK Vishnu/Paul about debugging** 

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*
Added Meditation Feature

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*
No tests yet

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by vishnuravi** [comment]
Hey @dguo8412 how are things going with this PR? What still needs to be done?

cc: @PSchmiedmayer @griffinac

**Comment by dguo8412** [comment]
I added alot more to the meditations feature. It is almost done.

File: Balance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,21 +7,18 @@
...
 		2720083A29A7E57D0052908D /* DiaryHomeView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2720083529A7E4C20052908D /* DiaryHomeView.swift */; };
 		2720083B29A7E57F0052908D /* DiaryNoteEntryView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2720083629A7E4C20052908D /* DiaryNoteEntryView.swift */; };
 		2720083D29A7E5830052908D /* PastDiaryEntry.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2720083829A7E4C20052908D /* PastDiaryEntry.swift */; };
 		2720084329A843B90052908D /* Note.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2720084229A843B90052908D /* Note.swift */; };
 		2720084B29A84A980052908D /* Date.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2720084A29A84A980052908D /* Date.swift */; };
-		276222BD29B0E3E100FCB4C5 /* NoteStore.swift in Sources */ = {isa = PBXBuildFile; fileRef = 276222BC29B0E3E100FCB4C5 /* NoteStore.swift */; };
-		2F39333729A8A5690035A05C /* FirebaseFirestore in Frameworks */ = {isa = PBXBuildFile; productRef = 2F39333629A8A5690035A05C /* FirebaseFirestore */; };
-		2761583B29A63BD4006C7AC1 /* SpotifyConstants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2761583A29A63BD4006C7AC1 /* SpotifyConstants.swift */; };
```
Comment (lowbaj21) [inline Balance.xcodeproj/project.pbxproj:18]: do we want to keep SpotifyConstants.swift here? this removes it from the PBXBuildFile section

File: Balance.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -80,20 +78,18 @@
...
 		2720083529A7E4C20052908D /* DiaryHomeView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DiaryHomeView.swift; sourceTree = "<group>"; };
 		2720083629A7E4C20052908D /* DiaryNoteEntryView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DiaryNoteEntryView.swift; sourceTree = "<group>"; };
 		2720083829A7E4C20052908D /* PastDiaryEntry.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PastDiaryEntry.swift; sourceTree = "<group>"; };
 		2720084229A843B90052908D /* Note.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Note.swift; sourceTree = "<group>"; };
 		2720084A29A84A980052908D /* Date.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Date.swift; sourceTree = "<group>"; };
-		276222BC29B0E3E100FCB4C5 /* NoteStore.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = NoteStore.swift; sourceTree = "<group>"; };
-		2761583A29A63BD4006C7AC1 /* SpotifyConstants.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = SpotifyConstants.swift; sourceTree = "<group>"; };
```
Comment (lowbaj21) [inline Balance.xcodeproj/project.pbxproj:84]: do we want to remove SpotifyConstants.swift here? I think we'd want it here too

File: Balance.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -352,7 +348,6 @@
 				2F9056F0299E3181003D3802 /* FirestoreDataStorage */,
 				2F9056F2299E3181003D3802 /* FirestoreStoragePrefixUserIdAdapter */,
 				2F9056F4299E3205003D3802 /* FHIRToFirestoreAdapter */,
-				5AAFCE1929A75990006CCD66 /* LocalStorage */,
```
Comment (lowbaj21) [inline Balance.xcodeproj/project.pbxproj:329]: do we want to keep LocalStroage instead of removing it?

File: Balance.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -192,7 +189,6 @@
 			children = (
 				2720083529A7E4C20052908D /* DiaryHomeView.swift */,
 				2720083629A7E4C20052908D /* DiaryNoteEntryView.swift */,
-				2720083429A7E4C20052908D /* HeaderMenu.swift */,
```
Comment (lowbaj21) [inline Balance.xcodeproj/project.pbxproj:183]: HeaderMenu contains views so I think we'd still want it in this Views section.

File: Balance.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -276,7 +272,7 @@
 		653A254F283387FE005D4D48 /* Balance */ = {
 			isa = PBXGroup;
 			children = (
-				4ACC786529AC39A40048D1D1 /* HeaderMenu.swift */,
```
Comment (lowbaj21) [inline Balance.xcodeproj/project.pbxproj]: I think this should contain other files:

children = (
				2720083329A7E4C20052908D /* BalanceDiaryFeature */,
				8900585629AC0DDA005339D9 /* BalanceDebug.entitlements */,
				2761584E29A662DE006C7AC1 /* Music */,
				653A2550283387FE005D4D48 /* Balance.swift */,
				2F5E32BC297E05EA003432F8 /* BalanceAppDelegate.swift */,
				2F4E23822989D51F0013F3D9 /* BalanceAppTestingSetup.swift */,
				2FC975A72978F11A00BA99FE /* Home.swift */,
				2FC9759D2978E30800BA99FE /* Supporting Files */,
				2761584429A64178006C7AC1 /* Balance-Bridging-Header.h */,
				2761584529A6448C006C7AC1 /* SceneDelegate.swift */,
			);

**Review by lowbaj21 — COMMENTED** [review]
may want to revisit organization of files in .pbxproj

**Review by lowbaj21 — APPROVED** [review]
comments addressed. looks good!
